### PR TITLE
Configurable `quiet_time` and `polling` for Celery tasks

### PR DESCRIPTION
**Pull request by alfredodeza**
Fixes a few issues with deployment as well that would prevent celery from working correctly.

### CODE REVIEW

File: chacra/async/__init__.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import pecan
 from celery import Celery
 import celery
+import time
```
Comment (andrewschoen) [inline chacra/async/__init__.py]: is this still needed?

File: config/dev.py [hunk 2]
```diff
@@ -70,5 +71,15 @@
...
 api_user = 'admin'
 api_key = 'secret'
+
+# Celery options
+# How often (in seconds) the database should be queried for repos that need to
+# be rebuilt
+polling_cycle = 15
+
+# Once a "create repo" task is called, how many seconds (if any) to wait before actually
+# creating the repository
```
Comment (andrewschoen) [inline config/dev.py]: maybe explain here why this is important? IIRC, it was to account for bulk binary uploads.

File: deploy/playbooks/roles/common/tasks/nginx.yml [hunk 3]
```diff
@@ -35,4 +35,4 @@
 
 - name: ensure nginx is started
```
Comment (andrewschoen) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: nitpick: should say "restart nginx" now.

File: chacra/async/__init__.py (hunk 1 above)
Comment (alfredodeza) [inline chacra/async/__init__.py]: no -_-

**Pull request merged**